config/locales/en.yml

```diff
@@ -169,6 +169,7 @@ en:
       list: "%{class}-class — a list rather than a conventional article"
       ?: Unrated — an article that hasn't yet been rated on Wikipedia's quality scale
       does_not_exist: This article does not exist on Wikipedia
+    references_doc: It is the number of citations added in the article.This data comes for the ORES article quality model, and is only available for some languages.
```

Comment:
"This is the number of reference tags added to articles, and can include multiple references to the same source. The data comes from the ORES article quality model, and is only available for some languages."